src/main/scala/cromwell/engine/backend/jes/JesBackend.scala

```diff
@@ -165,13 +164,13 @@ class JesBackend extends Backend with LazyLogging {
 
   def generateJesInputs(backendCall: BackendCall): Iterable[JesInput] = {
     val adjustedPaths = adjustInputPaths(backendCall.call, backendCall.locallyQualifiedInputs)
-    def mkInput(k: String, v: WdlFile) = JesInput(k, backendCall.lookupFunction(k).valueString, Paths.get(v.valueString))
+    def mkInput(fileTag: String, location: WdlFile) = JesInput(fileTag, backendCall.lookupFunction(fileTag).valueString, Paths.get(location.valueString))
     adjustedPaths collect {
-      case (k: String, v: WdlFile) =>
-        logger.info(s"${makeTag(backendCall)}: $k -> ${v.valueString}")
-        Seq(mkInput(k, v))
-      case (k: String, v: WdlArray) => v.value.collect { case f: WdlFile => mkInput(k, f) }
-      case (k: String, v: WdlMap) => v.value flatMap { case (k, v) => Seq(k, v) } collect { case f: WdlFile => mkInput(k, f) }
+      case (fileTag: String, location: WdlFile) =>
+        logger.info(s"${makeTag(backendCall)}: $fileTag -> ${location.valueString}")
+        Seq(mkInput(fileTag, location))
+      case (fileTag: String, location: WdlArray) => location.value.collect { case f: WdlFile => mkInput(fileTag, f) }
+      case (fileTag: String, location: WdlMap) => location.value flatMap { case (k, v) => Seq(k, v) } collect { case f: WdlFile => mkInput(fileTag, f) }
```

Comment:
NB this change is nothing to do with the rest - it fixes a "suspicious variable name shadowing pattern" warning of k's and v's.